pcmdi_metrics/misc/scripts/parallelize_driver.py

```diff
@@ -2,11 +2,11 @@
 from __future__ import print_function
```

Comment:
```suggestion
```

`from __future__ import print_function` probably exists throughout the codebase. I don't think it'll break anything to keep around, but it is best to remove as this is only needed for Python 2.